frontend/pages/DashboardPage/_styles.scss

```diff
@@ -24,10 +24,7 @@
   }
 
   &__header {
-    height: 38px;
```

Comment:
In [frontend/styles/var/mixins.scss](https://github.com/fleetdm/fleet/pull/25182/files#diff-ec964e0ccf27de0e39bf2d3cab713a007abf605b8a3a3e5c9c33fe68b051b064), frontend keeps a lot of our reusable styling . I noticed we have this same:
```
  display: flex;
  align-items: center;
  height: 38px;
  justify-content: space-between;
```
over and over in our `*.scss` style sheets so I replaced them all with a new mix-in